u-boot-tools/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bin Meng <bmeng.cn@gmail.com>
+
+pkgname=u-boot-tools
+pkgver=2020.01
+pkgrel=1
+pkgdesc="U-Boot Tools"
+arch=('i686' 'x86_64')
+url="https://github.com/u-boot/u-boot"
+license=('GPL2')
+groups=('base')
+depends=('openssl')
+makedepends=('unzip' 'openssl-devel')
+source=("https://github.com/u-boot/u-boot/archive/v${pkgver}.zip")
```

Comment:
Downloadd in named archive

Reply:
Thanks for the review. Code changed and pushed.